app/ts/components/pages/AddNewAddress.tsx

```diff
@@ -330,6 +331,28 @@ export function AddNewAddress(param: AddAddressParam) {
 		})
 	}
 
+	function showOnChainVerificationErrorBox() {
+		return incompleteAddressBookEntry.entrySource === 'OnChain' && (incompleteAddressBookEntry.type === 'ERC20' || incompleteAddressBookEntry.type === 'ERC721')
+	}
+
+	function isSubmitButtonDisabled() {
+		return !areInputValid()
```

Comment:
just minor grammar with `are` implying many. either `isInputValid` or `areInputsValid` for clarity

Reply:
`input` is also plurar for `input` https://grammarhow.com/input-or-inputs/